verification/curator-service/ui/src/components/new-case-form-fields/Location.tsx

```diff
@@ -1,5 +1,6 @@
 import { Typography, withStyles } from '@material-ui/core';
 
+import { Location as CaseLocation } from '../Case';
```

Comment:
Maybe `Location as Loc` to keep in sync with other files

Reply:
sounds good, done.